eng/Versions.props

```diff
@@ -82,7 +82,7 @@
     <CoverletConsolePackageVersion>1.5.0</CoverletConsolePackageVersion>
     <DotNetReportGeneratorGlobalToolPackageVersion>4.1.4</DotNetReportGeneratorGlobalToolPackageVersion>
     <!-- Docs -->
-    <MicrosoftPrivateIntellisensePackageVersion>3.0.0-preview9-190909-1</MicrosoftPrivateIntellisensePackageVersion>
+    <MicrosoftPrivateIntellisensePackageVersion>3.1.0-preview2-191021-1</MicrosoftPrivateIntellisensePackageVersion>
```

Comment:
We never produced a stable version for 3.0, probably because it wasn't necessary?

Reply:
Are you asking if we generated a version for GA?
If that's what you meant, then yes, we did. Documentation is always one version behind, so what went into GA is what got introduced in Prev9. [Here's the PR ](https://github.com/dotnet/corefx/pull/40965) where I merged that.

Reply:
I was asking if we should have generated a stable nuget package for the documentation for 3.0.

Reply:
I am not sure what you mean by stable nuget package. That's why I thought you were talking about 3.0 GA (stable) vs Previews (unstable).
Can you please elaborate?

Reply:
`3.0.0-preview9-190909-1` is a non stable version. This is flagged by nuget as a prerelease package. My question was if it's fine that we are using a prerelease package in the GA 3.0 branch.

Reply:
I don't think it matters that it is flagged, AFAIK, because this package is only consumed internally by our repo to download the English intellisense files and puts them in the right location when necessary. Customers don't consume this.

Your comment just made me realize that because we are about to ship Prev2, this nupkg needed to be named Prev1. Because, as I said in my first comment, we are always one version behind. @ahsonkhan can you please confirm what I just said?

Reply:
Transport packages like this don't need to stabilize.  It's actually an added safegaurd that they don't since it prevents them from ever leaking into the shipping package graph.

Reply:
> can you please confirm what I just said?

Yes. What version of the dlls are the docs written against? If it's what we shipped as part of 3.1 preview 1, then yes, the intellisense package should be for preview 1 (i.e. lag behind by one version). If we shared the nightly preview 2 bits with docs, then it would be preview 2 (but I don't believe that's the case).